k8s/manifests/kube-prometheus/lib/postgres.libsonnet

```diff
@@ -10,6 +10,16 @@ local postgresMixin = addMixin({
 {
   grafanaDashboards: postgresMixin.grafanaDashboards,
   prometheusRules: postgresMixin.prometheusRules,
-  exporterService: common.exporterService('postgres', 9187, 'simple-v1'),
-  serviceMonitor: common.serviceMonitor('postgres', 'simple-v1'),
-}
+  exporterServices: [
+    common.exporterService('postgres', 9187, 'simple-v1'),
+    common.exporterService('postgres', 9187, 'dhis2-demo-ecuador'),
```

Comment:
Since all the services have the same label, these services will be redundant, listening the the same set of instances.